packages/hooks/use-lockscreen/index.ts

```diff
@@ -38,10 +37,6 @@ export const useLockscreen = (
 
   const hiddenCls = computed(() => ns.bm('parent', 'hidden'))
 
-  if (!isClient || hasClass(document.body, hiddenCls.value)) {
```

Comment:
In order to prevent errors in non-browser environments, the isClient judgment is necessary.

Reply:
@btea cc